README.md

```diff
@@ -75,123 +75,6 @@ it('works before and after upgrading', async function () {
 });
 ```
 
-## Usage
-
-
-## How do the plugins work?
```

Comment:
I think we want to keep some content like this on the readme. The links can go to the FAQ on the docs site. But feel free to also just remove the links for now until we know what the actual links will be when we publish.

Reply:
I have added How to the plugins work and Managements back in but without links to the FAQ.

Reply:
If we know what the URL will be, I can add the links to the API and to the FAQ.

Reply:
I added links to the FAQ, and for each plugin a link to the API and the main docs.